### PR TITLE
Add zoom and fix input label & placeholder

### PR DESCRIPTION
**Pull request by edellucien**
- The label "Tile URL" laps out of the window (all browsers). It must be placed closer to the input field.
 - The label should not say "Tiles URL" but "GL Style URL"
 - place a hint in the field, saying e.g. "Paste the URL to a GL Style JSON here" and set empty string as default tiles url
 - add +/- zoom buttons

### CODE REVIEW

File: src/components/Map/Map.js [hunk 1]
```diff
@@ -67,23 +74,34 @@ const Map = ({ permalinkParam }) => {
...
+          }}
+        />
+      </div>
     </>
   );
 };
 
 Map.propTypes = {
   permalinkParam: PropTypes.string,
+  map: PropTypes.instanceOf(OLMap),
```
Comment (oterral) [inline src/components/Map/Map.js:99]: Why are you using a map we don't need it

Comment (oterral) [reply src/components/Map/Map.js:99]: ah forget you need it for the zoom

File: src/App.js [hunk 2]
```diff
@@ -14,14 +17,23 @@ const useStyles = makeStyles(() => ({
...
   return (
     <Grid container className={classes.container}>
       <Grid item xs={6} className={classes.mapLeft}>
         <Map permalinkParam="left" />
       </Grid>
       <Grid item xs={6} className={classes.mapRight}>
-        <Map permalinkParam="right" />
+        <Map permalinkParam="right" map={map} />
       </Grid>
+      <Zoom map={map} />
```
Comment (oterral) [inline src/App.js:36]: May be one Zoom for each map. Ask uli

File: src/components/Map/Map.js [hunk 3]
```diff
@@ -67,23 +74,34 @@ const Map = ({ permalinkParam }) => {
...
+          onFocus={() => onFocused(true)}
+          onBlur={() => onFocused(false)}
+          placeholder="Paste the URL to a GL Style JSON here"
+          variant="outlined"
+          value={tilesUrl}
+          onChange={(evt) => {
+            setTilesUrl(evt.target.value);
+          }}
+        />
+      </div>
```
Comment (oterral) [inline src/components/Map/Map.js:92]: I dont'think you need the form label
```
<TextField
          id="outlined-full-width"
          label="Label"
          style={{ margin: 8 }}
          placeholder="Placeholder"
          fullWidth
          margin="normal"
          InputLabelProps={{
            shrink: true,
          }}
          variant="outlined"
        />
```

**Review by oterral — CHANGES_REQUESTED** [review]
Simplify the css and components

File: src/components/Map/Map.js (hunk 3 above)
Comment (edellucien) [reply src/components/Map/Map.js:92]: if you pass the label, it will overlap with the placeholder.
This way we are allowed to have both

Comment (oterral) [reply src/components/Map/Map.js:92]: Did you test what I send you ?

Comment (edellucien) [reply src/components/Map/Map.js:88]: Yes I changed the code with the inspiration opf your example thanks :+1:

File: src/App.js (hunk 2 above)
Comment (edellucien) [reply src/App.js]: Uli asked for a zoom per map. I changed it

**Review by oterral — CHANGES_REQUESTED** [review]
Can you convert sass file to scss please.

**Review by oterral — APPROVED** [review]

**Pull request merged**